spec/models/product_spec.rb

```diff
@@ -128,6 +128,15 @@
     end
   end
 
+  describe '#friendly_name' do
+    let(:product_attrs) { { name: 'Dummy Product', friendly_version: '99', release_type: 'Bar', arch: 'x86_64' } }
```

Comment:
This doesn't actually test for what you have fixed. I know you are working on the serializer spec also, but why not throw in a `release_stage: "bazinga!"` and keep the rest as is. Then this test would fail if anybody every added the release stage back to the friendly name generation.